Cargo.toml

```diff
@@ -21,6 +21,7 @@ client-server = []
 ansi_colours = "1.0"
 bet = "0.3.4"
 clap = { version="2.33", default-features=false, features=["suggestions"] }
+cli-clipboard = {path="/home/zx/workspace/git/cli-clipboard"}
```

Comment:
I looked at a lot of crates and this was simplest in that it didn't follow "rust dogmatism", no need to create a clipboard instance (because system clipboards are as global as they come) etc.

Reply:
Why did you link to a local copy ? I changed that to `cli-clipboard = "0.2"` and it looks fine

Reply:
Sorry I left that in by mistake.